### PR TITLE
Added NCache Live in Customers & Partners section in README.md

### PR DESCRIPTION
**Pull request by Aussama-Ul-Islam**
NCache Live by Alachisoft is a web application which lets user use NCache without installing and setting up it in their environment. There is a pre-configured environment present for user. All he need to do is to access the website and start using NCache API. We have used Try .NET to make execution of code snippets and editing them possible.

### CODE REVIEW

**Comment by Aussama-Ul-Islam** [comment]
I have done the PR and waiting for response or approval. The only change I have made is in Customers and Partners section of README.md file.

File: README.md
Comment (LadyNaggaga) [inline README.md:47]: Add a heading above titled **Community Showcase**

Comment (Aussama-Ul-Islam) [reply README.md:49]: I have made the required changes. Please have a look at them.

**Comment by Aussama-Ul-Islam** [comment]
@LadyNaggaga Thank you for your response. I have made the changes as per your requirements. Please review them.

**Comment by Aussama-Ul-Islam** [comment]
I just wanted to follow up on this PR. I have made the required changes and waiting for this PR to get reviewed and merged so that others can see the power of Try .NET with the help of NCache Live.

**Review by jonsequitur — APPROVED** [review]

**Comment by jonsequitur** [comment]
Thanks, @Aussama-Ul-Islam!

**Pull request merged**
